src/authentication.ts

```diff
@@ -64,9 +65,10 @@ export const fetchTokens = (config: TInternalConfig): Promise<TTokenResponse> =>
     a 'code' url parameter.
     This code will now be exchanged for Access- and Refresh Tokens.
   */
+  const storage = config.storage === 'session' ? sessionStorage : localStorage
```

Comment:
Sorry for not spotting this one previously, but this will of course also need to be reversed.

```suggestion
  const storage = config.storage === 'local' ? localStorage : sessionStorage
```